app/styles/pages/bulletin-form.scss

```diff
@@ -0,0 +1,8 @@
+.bulletin-form {
+  .banner-preview {
+    height: 200px;
```

Comment:
Properties should be ordered background-repeat, background-size, height, margin-bottom